commitizen/commands/check.py

```diff
@@ -79,15 +83,15 @@ def __call__(self):
 
     def _get_commits(self):
         # Get commit message from file (--commit-msg-file)
-        if self.commit_msg_file:
+        if self.commit_msg_file is not None:
             with open(self.commit_msg_file, "r", encoding="utf-8") as commit_file:
                 msg = commit_file.read()
             msg = self._filter_comments(msg)
             msg = msg.lstrip("\n")
             commit_title = msg.split("\n")[0]
             commit_body = "\n".join(msg.split("\n")[1:])
             return [git.GitCommit(rev="", title=commit_title, body=commit_body)]
-        elif self.commit_msg:
+        elif self.commit_msg is not None:
```

Comment:
Same here

Reply:
In that case, could you please add a comment here. I think we might miss it in the future

Reply:
Good idea; done. Also, removing `is not None` on this line (but not the one above) will currently cause tests to fail.